ocdsdata/maindatabase/migrations/env.py

```diff
@@ -11,9 +10,8 @@
 
 config.set_main_option("sqlalchemy.url", ocdsdata.maindatabase.config.DB_URI)
 
-# Interpret the config file for Python logging.
-# This line sets up loggers basically.
-fileConfig(config.config_file_name)
+# We do NOT set up logging here; our command line tool already does that.
```

Comment:
Is this comment necessary? Design principles typically live in a documentation file.

Reply:
I just thought seeing as that is the standard place you would expect to see some logging stuff in an Alembic env file, it's good to put it there for anyone going "Wait, why aren't we doing logging here as usual?". But not fussed if others don't want it there.

Reply:
Ah, yes I see it's a standard block: https://github.com/zzzeek/alembic/blob/master/alembic/templates/generic/env.py

We can leave the comment.